samples/interceptors/wso2-micro-integrator/README.md

```diff
@@ -0,0 +1,69 @@
+# Sample Interceptor - WSO2 Micro Integrator
+
+The interceptor service is generated using the interceptor service Open API definition.
+[Swagger Editor](https://editor.swagger.io/) is used to generate the Synapse API configuration file in Micro Integrator.
+
+NOTE: You need to convert the open-api 3.0.0 to swagger 2.0, to sucessfully add a new REST API in Micro Integrator.
+
+To learn about interceptors and for instructions on how to run it please head over to the
+[home directory of interceptor samples](../README.md#samples).
+
+## Prerequisites
+- [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/)
+- Docker
+- Java 11
+- Maven
```

Comment:
Should we need a specific maven version like maven 3.6 ?

Reply:
there wasn't a specified version mentioned in the repo (https://github.com/wso2/integration-studio). It worked with  3.5.4 which I have installed.